### PR TITLE
CORE-V: SIMD Update: cv.or.sci[.h,.b], cv.xor.sci[.h,.b], cv.and.sci[.h,.b], cv.avgu.sci[.h,.b]

### PR DESCRIPTION
**Pull request by NandniJamnadas**
Updated cv.or.sci[.h,.b], cv.xor.sci[.h,.b], cv.and.sci[.h,.b] from unsigned to signed.
Updated cv.avgu.sci[.h,.b] from signed to unsigned.

	* opcodes/riscv-opc.c: Updated cv.or.sci[.h,.b],
	  cv.xor.sci[.h,.b], cv.and.sci[.h,.b] from unsigned to signed.
	  Updated cv.avgu.sci[.h,.b] from signed to unsigned.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-b-fail.l: Updated tests.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-b-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-b-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-b-pass.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-h-fail.l: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-h-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-h-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-and-sci-h-pass.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-b-fail.l: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-b-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-b-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-b-pass.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-h-fail.l: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-h-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-h-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-avgu-sci-h-pass.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-b-fail.l: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-b-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-b-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-b-pass.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-h-fail.l: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-h-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-h-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-or-sci-h-pass.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-b-fail.l: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-b-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-b-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-b-pass.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-h-fail.l: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-h-fail.s: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-h-pass.d: Likewise.
	* gas/testsuite/gas/riscv/cv-simd-xor-sci-h-pass.s: Likewise.

### CODE REVIEW

**Comment by NandniJamnadas** [comment]
This pull request fixes CORE-V GCC issue [#49](https://github.com/openhwgroup/corev-gcc/issues/49) Toolchain flags errors for xpulp SIMD ALU instuctions cv.or.sci, cv.xor.sci, cv.and.sci with negative (signed) Imm6 operand.

**Comment by NandniJamnadas** [comment]
This pull request fixes CORE-V GCC issue [#51](https://github.com/openhwgroup/corev-gcc/issues/51) Toolchain flags errors for xpulp SIMD ALU instuctions cv.avgu.sci.{.h/.b} with 6-bit unsigned decimal notation for Imm6 operand

**Review by jeremybennett — APPROVED** [review]
This is a fairly routine change to sync the instructions with the specification.  Good to merge.

File: gas/testsuite/gas/riscv/cv-simd-and-sci-b-fail.s [hunk 1]
```diff
@@ -2,6 +2,6 @@ target:
     #Boundary Register Tests
     cv.and.sci.b x32, x32, 20
     cv.and.sci.b x33, x33, 20
-    #Boundary Immediate Values Test
```
Comment (simonpcook) [inline gas/testsuite/gas/riscv/cv-simd-and-sci-b-fail.s:5]: FYI this typo snuck through

**Pull request merged**